libse/Dictionaries/OcrFixReplaceList.cs

```diff
@@ -113,14 +113,19 @@ public static OcrFixReplaceList FromLanguageId(string languageId)
                 XmlNode node = doc.DocumentElement.SelectSingleNode(name);
                 if (node != null)
                 {
+                    const string ReplaceWith = "replaceWith";
+                    const string Find = "find";
                     foreach (XmlNode item in node.ChildNodes)
                     {
-                        if (item.Attributes != null && item.Attributes["replaceWith"] != null && item.Attributes["find"] != null)
+                        if (item.Attributes != null && item.Attributes[ReplaceWith] != null && item.Attributes[Find] != null)
                         {
-                            string to = item.Attributes["replaceWith"].InnerText;
-                            string from = item.Attributes["find"].InnerText;
-                            if (!list.ContainsKey(from))
-                                list.Add(from, to);
+                            string to = item.Attributes[ReplaceWith].InnerText;
+                            string from = item.Attributes[Find].InnerText;
```

Comment:
[Windows PowerShell in Action, Second Edition](https://www.manning.com/books/windows-powershell-in-action-second-edition)

It's about V2, hence, it doesn't address V3/V4 additions, but it's the best posh reference I know. It starts every subject simple (no previous knowledge required), before going into more complex details, until everything is revealed. :smile: 

It's miles ahead of other posh books I read, only downside is that there have been no upgrades for V3 and V4. Of course, MSDN contains info on changes between V2-V3, and V3-V4. And for the latest *usage* of any cmdlet, you should use `get-help -online` anyway.